templates/public/bundle.js

```diff
@@ -77,7 +77,7 @@
 /******/ 	__webpack_require__.o = function(object, property) { return Object.prototype.hasOwnProperty.call(object, property); };
 /******/
 /******/ 	// __webpack_public_path__
-/******/ 	__webpack_require__.p = "/Users/ssuresh/Desktop/fun-night/templates/public";
+/******/ 	__webpack_require__.p = "/Users/shubham/repo/fun-night/templates/public";
```

Comment:
This should be computer agnostic

Reply:
@ssuresh2dev what should this be

Reply:
i think this file should be in the .gitignore lol it gets autogenerated when you run npm scripts

Reply:
it's just a compiled version of all the React code into a single bundle so that it's easier for the browser to download